android/app/src/test/java/org/ocua/parity/BookkeeperTest.java

```diff
@@ -21,14 +23,16 @@ public class BookkeeperTest {
     private Bookkeeper bookkeeper;
     private Team home;
     private Team away;
+    private League league;
 
     @Before
     public void before() {
         int week = 1;
         home = new Team("Team A", 1);
         away = new Team("Team B", 2);
+        league = new League();
         bookkeeper = new Bookkeeper();
-        bookkeeper.startGame(week, home, away);
+        bookkeeper.startGame(league, week, home, away);
```

Comment:
I guess this was just broken? we don't run these tests automatically so makes sense

Reply:
Yeah, the `League` argument was added after the unit test was last updated